src/dev.sh

```diff
@@ -361,7 +361,16 @@ bash ./Misc/externals.sh $RUNTIME_ID "Pre-Cache" || checkRC "externals.sh Pre-Ca
 
 if [[ "$CURRENT_PLATFORM" == 'windows' ]]; then
     vswhere=$(find "$DOWNLOAD_DIR" -name vswhere.exe | head -1)
-    vs_location=$("$vswhere" -latest -property installationPath)
+
+    # Tim Brigham - update to the default string first to search 
+    vs_location=$("$vswhere"  -latest -property installationPath)
+
+    # Then only if it is empty perform a second search wiith the BuildTools product set 
+    if [[ ! -e "${vs_location}" ]]; then
+        #try again with -products 'Microsoft.VisualStudio.Product.BuildTools' set 
+        vs_location=$("$vswhere"  -latest -property installationPath -products 'Microsoft.VisualStudio.Product.BuildTools')
+    fi
+
```

Comment:
Are these changes related to CLM support? 
If not, I would move this to a separate PR.

Reply:
I've disabled this change, will enter a separate PR to address in the future.